### PR TITLE
Farnaz

### PR DESCRIPTION
**Pull request by Farnazmdi**

### CODE REVIEW

File: lineage/figures/Fig_Gen.py [hunk 1]
```diff
@@ -124,8 +127,9 @@ def Lineage_Length(T_MAS=500, T_2=100, reps=10, MASinitCells=[1], MASlocBern=[0.
                 X, newLineage, masterLineage, subLineage2 = Breadth_Two_State_Lineage(
                     experimentTime=T_MAS + T_2, initCells=MASinitCells, locBern=MASlocBern, betaExp=MASbeta, switchT=T_MAS, bern2=locBern2, betaExp2=beta2, FOM=FOM, verbose=False)
 
-        X = remove_unfinished_cells(X)
+#         X = remove_unfinished_cells(X)
```
Comment (aarmey) [inline lineage/figures/Fig_Gen.py]: Why not just remove the line?

File: lineage/figures/Fig_Gen.py [hunk 2]
```diff
@@ -235,12 +240,15 @@ def Lineages_per_Population_Figure(lineage_start=1, lineage_end=2, numStates=2,
                         X, newLineage, masterLineage, subLineage2 = Breadth_Two_State_Lineage(
                             experimentTime=T_MAS + T_2, initCells=MASinitCells, locBern=MASlocBern, betaExp=MASbeta, switchT=T_MAS, bern2=locBern2, betaExp2=beta2, FOM=FOM, verbose=False)
 
-                X = remove_unfinished_cells(X)
+#                 X = remove_unfinished_cells(X)
```
Comment (aarmey) [inline lineage/figures/Fig_Gen.py]: Same here.

File: lineage/figures/Fig_Gen.py [hunk 3]
```diff
@@ -235,12 +240,15 @@ def Lineages_per_Population_Figure(lineage_start=1, lineage_end=2, numStates=2,
...
                 X = remove_singleton_lineages(X)
+                X, end_time = select_population(X, experimentTime)
+                newLineag, end = select_population(newLineage, experimentTime)
                 X1.extend(newLineage)
 
-            X1 = remove_unfinished_cells(X1)
+#             X1 = remove_unfinished_cells(X1)
```
Comment (aarmey) [inline lineage/figures/Fig_Gen.py]: And here.

File: lineage/figures/Fig_Gen.py (hunk 1 above)
Comment (Farnazmdi) [reply lineage/figures/Fig_Gen.py]: I will remove them.

**Comment by Farnazmdi** [comment]
@aarmey Sorry for requesting the review again, I reopened the PR and it doesn't let me merge without a review.

**Pull request merged**